.vscode/spellright.dict

```diff
@@ -0,0 +1,3 @@
+analog_controller
```

Comment:
This file is not needed on the server and can be removed from the next commit.

Reply:
oh ya it is a mistake .... it is dictionary file